ovirt_imageio/_internal/auth.py

```diff
@@ -286,18 +291,20 @@ def cancel(self, timeout=60):
 
         with self._lock:
             self._canceled = True
-            if not self._connections:
+            if not self._ongoing:
```

Comment:
When ticket is canceled, we cannot close connections blocked on client socket
but we can and should close the connection context, that will close the backend.

If we don't close the backend, deactivating logical volumes will fail when using file
backend, since imageio keeps an open file descriptor on the logical volume.

We need to remove and close all connections contexts.